tests/Operation/BulkWriteTest.php

```diff
@@ -57,7 +57,7 @@ public function testInsertOneDocumentArgumentMissing(): void
     public function testInsertOneDocumentArgumentTypeCheck($document): void
     {
         $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessageMatches('/Expected \$operations\[0\]\["insertOne"\]\[0\] to have type "array or object" but found "[\w ]+"/');
+        $this->expectExceptionMessageMatches('/Expected \$operations\[0\]\["insertOne"\]\[0\] to have type "document" but found ".+"/');
```

Comment:
I don't recall why we accepted a space in the original pattern.

Reply:
"I'm sure it seemed like a good idea at the time" 😅